extensions/amp-sidebar/1.0/component.js

```diff
@@ -166,11 +167,81 @@ function SidebarWithRef(
         >
           <div className={classes.backdropOverscrollBlocker}></div>
         </div>
-      </>
-    )
+      </div>
+    </>
   );
 }
 
 const Sidebar = forwardRef(SidebarWithRef);
 Sidebar.displayName = 'Sidebar'; // Make findable for tests.
 export {Sidebar};
+
+/**
+ * @param {!SidebarDef.SidebarNavToolbar} props
+ * @return {PreactDef.Renderable}
+ */
+export function SidebarNavToolbar({
```

Comment:
Opinions on this name?  I am open to suggestions.  It would look something like this in Preact mode:

```
<Sidebar>
        <span>
                Lorem ipsum dolor sit amet, has...
        </span>
        <SidebarNavToolbar toolbar="(max-width: 767px)" toolbar-target="target-element">
                My Nav Toolbar Content...
        </SidebarNavToolbar>
<Sidebar/>
```

Reply:
Ya, I like `SidebarToolbar`, let's go with that.  The `Nav` part is not crucial in my opinion.